modules/ftrackIdSystem.md

```diff
@@ -0,0 +1,90 @@
+# Flashtalking's FTrack Identity Framework User ID Module
+
+*The FTrack Identity Framework User ID Module allows publishers to take advantage of Flashtalking's FTrack ID during the bidding process.*
+
+**THE ONLY COMPLETE SOLUTION FOR COOKIELESS MEASUREMENT & PERSONALIZATION**
+
+Flashtalking is the world’s first ad serving platform to function without cookies to orchestrate client identity across buy-side ID spaces for measurement and personalization. With over 120 active global advertisers, our cookieless identity framework is market-ready and includes privacy controls to ensure consumer notification and choice on every impression.
+
+### [FTrack](https://www.flashtalking.com/identity-framework#FTrack)
+
+Flashtalking’s cookieless tracking technology uses probabilistic device recognition to derive a privacy-friendly persistent ID for each device.
+
+**PROVEN**  
+With over 120 brands using FTrack globally, Flashtalking has accumulated the largest cookieless footprint in the industry.
+
+**ANTI-FINGERPRINTING**  
+FTrack operates in strict compliance with [Google’s definition of anti-fingerprinting](https://blog.google/products/ads-commerce/2021-01-privacy-sandbox/). FTrack does not access PII or sensitive information and provides consumers with notification and choice on every impression. We do not participate in the types of activities that most concern privacy advocates (profiling consumers, building audience segments, and/or monetizing consumer data).
+
+**GDPR COMPLIANT**  
+Flashtalking is integrated with the IAB EU’s Transparency & Consent Framework (TCF) and operates on a Consent legal basis where required.  As a Data Processor under GDPR, Flashtalking does not combine data across customers nor sell data to third parties.
+
+**ACCURATE**  
+FTrack’s broad adoption combined with the maturity of the models (6+ years old) gives Flashtalking the global scale with which to maintain a high degree of model resolution and accuracy.
+
+**DURABLE**  
+As new IDs start to proliferate, they will serve as new incremental signals for our models.
```

Comment:
this is very borderline to being marketing language....please focus on specific facts about how your module works and let your website deal with pitching it.

Reply:
Removed